runtimes/runtimes/operational-telemetry/aws-metrics-exporter.ts

```diff
@@ -105,16 +105,20 @@ export class AwsMetricExporter implements PushMetricExporter {
 
         const unixEpochSeconds = metric.dataPoints[0].endTime[0]
         const result = {
-            name: metric.descriptor.name,
-            timestamp: unixEpochSeconds,
-            ...dataPoints,
+            baseInfo: {
+                name: metric.descriptor.name,
+                timestamp: unixEpochSeconds,
+            },
+            resourceValues: {
+                ...dataPoints,
+            },
         }
 
         const isValid = this.eventValidator.validateEvent(result)
         if (!isValid) {
             throw Error(`Invalid operational event: ${result}`)
         }
 
-        return result as OperationalEvent
+        return result as any
```

Comment:
Why was this change needed?

Reply:
We no longer accept `[k: string]: unknown;` parameters in the generated type, as a result of adding `"additionalProperties": false` to JSON schemas. When we fill the `result` event from a `Record<string, number> `object, TypeScript cannot verify its correctness at compile time.